### DISCUSSION TITLE
Rename repo polls

### DISCUSSION THREAD

**Discussion by black7375** [black7375/Firefox-UI-Fix#525] [Polls]
It has pointed out that the URL name and the project name are different. #514

This project was made for personal use, but was later branding.
(For historical reasons, it currently has a repository name)

I would like to vote about whether it is necessary to change.

**Comment by black7375** [comment]
You may suggest another name.

**Comment by Nomes77** [comment]
Since there are some many extra option, I wouldn't say it is one theme. Therefore, I found the name `Firefox-UI-Fix` better. It is now more then only a theme.

**Reply by JohnyP36** [reply]
`Firefox-Lepton-Theme` suggest that this repo is only for `lepton`. But I used it to make Firefox look like Microsoft Edge.  
So keep the original name. That will also save you time changing all the url's and other stuff  *- as @black7375 said before*.

**Reply by black7375** [reply]
[`Lepton`](https://en.wikipedia.org/wiki/Lepton) is a family name of [`Photon`](https://en.wikipedia.org/wiki/Photon) and [`Proton`](https://en.wikipedia.org/wiki/Proton), and it is a name for intended to be a light theme layer.

I plan to write more details at the end of this year or early next year.
This is [[Article] 0. Firefox UI UX history](https://github.com/black7375/Firefox-UI-Fix/wiki/%5BArticle%5D-0.-Firefox-UI-UX-history)'s follow-up article.

**Comment by Mertsch** [comment]
I think including `Theme` in the repo name is very good.
This it what someone without prior knowledge would type into awesomebar (and today find nothing)
firefox is clear, but too generic

`firefox theme` and get the correct suggestion

I am not so fond of `lepton`, as I use `photon`. But I have no better idea. Maybe something like "pre-Proton" theme

**Reply by black7375** [reply]
I wouldn't say I prefer anything to keep a value-neutral position in the vote, but I wouldn't use `pre-proton`.

This project has a goal of improving `proton`, so it should be matched.

> ":fox_face: I respect proton UI and aim to improve it. "

**Comment by black7375** [comment]
[`v7.0.0` has been released](https://github.com/black7375/Firefox-UI-Fix/releases/tag/v7.0.0) and maintains its current name.
![image](https://user-images.githubusercontent.com/25581533/210136660-a221491e-909a-4d87-a9bd-9b00e2fb6c0e.png)

**Reply by Nomes77** [reply]
As we are a week further, the current status is even better for keeping the name:
![afbeelding](https://user-images.githubusercontent.com/80090789/211012301-463cdfdf-669a-48b9-81c9-934f5d745f54.png)
